cmd/snap/cmd_interfaces.go

```diff
@@ -68,18 +68,18 @@ func (x *cmdInterfaces) Execute(args []string) error {
 			if x.Positionals.Query.Snap != "" && x.Positionals.Query.Snap != plug.Snap {
 				continue
 			}
-			if x.Positionals.Query.Name != "" && x.Positionals.Query.Name != plug.Name {
+			if x.Positionals.Query.Name != "" && x.Positionals.Query.Name != plug.Plug.Name {
```

Comment:
This is fixed with the upcoming /2.0/interfaces API change. It actually returns ``InterfaceConnections`` then but you'll have to wait a sec to see it.